src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-router.ts

```diff
@@ -62,6 +115,17 @@ class ZWaveJSConfigRouter extends HassRouterPage {
       );
     }
   }
+
+  private _loadData() {
+    if (this._unsubs) {
+      return;
+    }
+    this._unsubs = [
+      subscribeDeviceRegistry(this.hass.connection, (entries) => {
```

Comment:
Once you subscribe, you don't have to listen to events to loadData etc. It should happen automatically.